pwem/data.py

```diff
@@ -898,8 +899,7 @@ def __init__(self, filename=None, pseudoatoms=False, **kwargs):
 class EMSet(Set, EMObject):
 
     def _loadClassesDict(self):
-        import pyworkflow.em as em
-        classDict = em.Domain.getObjects()
+        classDict = Domain.getObjects()
```

Comment:
There is an even more general function Domain.getMapperDict, that should be used for setting the mapper dict. 
With this new approach, the basic objects defined in objects.py are already there. So one can remove the `from pyworkflow.object import *` at the top here or the globals()